### PR TITLE
SETI-498: CLI for credentials management

### PR DESCRIPTION
**Pull request by mezis**
This introduces `rtm`, a command-line wrapper for the Routemaster API.

===

Jira story [#SETI-498](https://deliveroo.atlassian.net/browse/SETI-498) in project *Software Engineering Tools and Infrastructure*:

> ## Why
> 
> Calling into APIs manually is cumbersome, and requires some back-and-forth with documentation and `curl` know-how.
> 
> ## What
> 
> Provide and `rtm` CLI tool to (in particular) manage credentials.

### CODE REVIEW

File: routemaster/client.rb [hunk 1]
```diff
@@ -183,7 +217,9 @@ def _send_event(event, topic, callback, t: nil, async: false, data: nil)
 
       def _check_pulse!
         _conn.get('/pulse').tap do |response|
-          raise 'cannot connect to bus' unless response.success?
+          # require 'pry' ; binding.pry unless response.success?
```
Comment (FriedSock) [inline routemaster/client.rb]: forgot this 😄

**Review by FriedSock — APPROVED** [review]

File: routemaster/cli/base.rb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          p.separator self.class.descr
+
+          if self.class.options
+            p.separator 'Options:'
+            self.class.options.call(p)
+          end
+
+          p.separator 'Common options:'
+
+          p.on('-b', '--bus DOMAIN|@NAME', %{
```
Comment (FriedSock) [inline routemaster/cli/base.rb:108]: What is the purpose of needing to prefix all names with `@`?

edit: Ah I see, anything could be a URI here. Realistically though; all URIs that will be used will contain a dot, so we could use that to distinguish between domains or names.

Comment (mezis) [reply routemaster/cli/base.rb:108]: Yeah, I think we could live without but it raises ambiguity.
(also - it's not URIs here, but domains, eg. `--bus rm.dev`. I'll add a note to clarify that - thanks!)

File: routemaster/client.rb (hunk 1 above)
Comment (mezis) [reply routemaster/client.rb]: D'OH.

**Pull request merged**